src/control/cmd/daos_agent/config_test.go

```diff
@@ -128,6 +140,10 @@ transport_config:
 			path:   badLogMaskCfg,
 			expErr: errors.New("not a valid log level"),
 		},
+		"bad filter config": {
+			path:   badFilterCfg,
+			expErr: errors.New("agent config validation failed"),
```

Comment:
I think it would be better to test for the specific error here ("cannot specify both exclude_fabric_ifaces and include_fabric_ifaces").

Reply:
Sure, I can do that.

Reply:
I lean toward the latter approach, as it verifies that we are actually testing what we think we are testing. I have definitely written unit tests that initially fail in a different location than I expected, because I overlooked some element of the setup.